fetch/cross-origin-resource-policy/resources/image.py

```diff
@@ -1,20 +1,22 @@
 import os.path
 
+from wptserve.utils import isomorphic_decode
+
 def main(request, response):
-    type = request.GET.first("type", None)
+    type = request.GET.first(b"type", None)
 
-    body = open(os.path.join(os.path.dirname(__file__), "green.png"), "rb").read()
+    body = open(os.path.join(os.path.dirname(isomorphic_decode(__file__)), u"green.png"), u"rb").read()
```

Comment:
This instance of `isomorphic_decode` seems odd. In my testing, `__file__` is a `str` in both py2 and py3. Is that not the case?

Reply:
There was a discussion at : 
https://github.com/web-platform-tests/wpt/pull/23771#discussion_r432121456

Reply:
@foolip, is the link above enough? Any more concerns?

Reply:
Sorry, I wasn't looking at my notifications. A bunch of recent changes have used `isomorphic_decode(__file__)` after #23771. As I tried to write up an issue about why I think that's not ideal, I convinced myself it's actually OK. In Python 3 there's the "Can't mix strings and bytes in path components" error you've probably run into, so we have to be consistent. https://docs.python.org/3.8/library/os.path.html explains how it's really a mess, but "Applications are encouraged to represent file names as (Unicode) character strings" which we can do, since we control all of the filenames.